src/core/Alert/Alert/Alert.md

```diff
@@ -0,0 +1,46 @@
+```js
+import { Alert, InlineAlert } from 'suomifi-ui-components';
+
+<>
+  <Alert closeText="Close" smallScreen>
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Alert>
+
+  <Alert closeText="Close" labelText="Maintenance notification">
```

Comment:
**Alert** interface does not have `labelText` as it was only for the **InlineAlert**.
So we need to remove it from this example.